src/coin/xtz/multisigUtils.ts

```diff
@@ -512,6 +512,20 @@ export function genericMultisigOriginationOperation(
   return originationOp;
 }
 
+/**
+ * Get the owners public key of an multisig wallet origination contract.
+ *
+ * @param {OriginationOp} operation An operation with the generic multisig wallet origination
+ * @returns {string[]} List of all the owners set in the origination transaction
+ */
+export function getOwnersPublicKey(operation: OriginationOp): string[] {
```

Comment:
readability-nit: use plural to denote multiple keys being returned

```suggestion
export function getOwnersPublicKeys(operation: OriginationOp): string[] {
```